### PR TITLE
Added changes to support for Firmware Drivers, Artifact Bundles

### PR DESCRIPTION
**Pull request by chebroluharika**
### Description
Added changes to support for Firmware Drivers, Artifact Bundles

### Issues Resolved
NA

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -377,16 +377,13 @@
...
 |<sub> /rest/artifact-bundles/backups/archive </sub>                              | POST             |  :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
 |<sub> /rest/artifact-bundles/backups/archive/{id} </sub>                         | GET              |  :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
 |<sub> /rest/artifact-bundles/backups/{id} </sub>                                 | GET              |  :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-|<sub> /rest/artifact-bundles/backups/{id} </sub>                                 | PUT              |  :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+|<sub> /rest/artifact-bundles/backups/{id} </sub>                                 | PUT              |  :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: ||
 |<sub> /rest/artifact-bundles/download/{id} </sub>                                | GET              |  :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
```
Comment (VenkateshRavula) [inline endpoints-support.md:386]: revert this change since it won't show any effect

Comment (chebroluharika) [reply endpoints-support.md:386]: done
